lib/ui/pointer.dart

```diff
@@ -258,46 +258,125 @@ class PointerData {
   final double scrollDeltaY;
 
   @override
-  String toString() => 'PointerData(x: $physicalX, y: $physicalY)';
+  String toString() => '$runtimeType(x: $physicalX, y: $physicalY)';
 
   /// Returns a complete textual description of the information in this object.
   String toStringFull() {
     return '$runtimeType('
-             'timeStamp: $timeStamp, '
-             'change: $change, '
-             'kind: $kind, '
-             'signalKind: $signalKind, '
-             'device: $device, '
-             'pointerIdentifier: $pointerIdentifier, '
-             'physicalX: $physicalX, '
-             'physicalY: $physicalY, '
-             'physicalDeltaX: $physicalDeltaX, '
-             'physicalDeltaY: $physicalDeltaY, '
-             'buttons: $buttons, '
-             'synthesized: $synthesized, '
-             'pressure: $pressure, '
-             'pressureMin: $pressureMin, '
-             'pressureMax: $pressureMax, '
-             'distance: $distance, '
-             'distanceMax: $distanceMax, '
-             'size: $size, '
-             'radiusMajor: $radiusMajor, '
-             'radiusMinor: $radiusMinor, '
-             'radiusMin: $radiusMin, '
-             'radiusMax: $radiusMax, '
-             'orientation: $orientation, '
-             'tilt: $tilt, '
-             'platformData: $platformData, '
-             'scrollDeltaX: $scrollDeltaX, '
-             'scrollDeltaY: $scrollDeltaY'
-           ')';
+        'timeStamp: $timeStamp, '
+        'change: $change, '
+        'kind: $kind, '
+        'signalKind: $signalKind, '
+        'device: $device, '
+        'pointerIdentifier: $pointerIdentifier, '
+        'physicalX: $physicalX, '
+        'physicalY: $physicalY, '
+        'physicalDeltaX: $physicalDeltaX, '
+        'physicalDeltaY: $physicalDeltaY, '
+        'buttons: $buttons, '
+        'obscured: $obscured, '
+        'synthesized: $synthesized, '
+        'pressure: $pressure, '
+        'pressureMin: $pressureMin, '
+        'pressureMax: $pressureMax, '
+        'distance: $distance, '
+        'distanceMax: $distanceMax, '
+        'size: $size, '
+        'radiusMajor: $radiusMajor, '
+        'radiusMinor: $radiusMinor, '
+        'radiusMin: $radiusMin, '
+        'radiusMax: $radiusMax, '
+        'orientation: $orientation, '
+        'tilt: $tilt, '
+        'platformData: $platformData, '
+        'scrollDeltaX: $scrollDeltaX, '
+        'scrollDeltaY: $scrollDeltaY'
+        ')';
```

Comment:
The indentation is to keep consistent with `lib/web_ui/lib/src/ui/pointer.dart` so it's easier to make `diff` compare.